tests/ui/macros/auxiliary/proc_macro_def.rs

```diff
@@ -1,4 +1,5 @@
 #![feature(proc_macro_quote)]
+#![feature(proc_macro_totokens)]
```

Comment:
Can this be avoided with `allow_internal_unstable`? The caller shouldn't need to enable `proc_macro_totokens` if just using the `quote!` macro. (They should still need to enable it if importing the trait and writing their own `ToTokens` impls for local types.)